Moblin/Various/Model.swift

```diff
@@ -245,7 +245,7 @@ final class Model: ObservableObject {
     private var obsWebSocket: ObsWebSocket?
     private var chatPostId = 0
     @Published var chatPosts: Deque<ChatPost> = []
-    private var pausedChatPosts: Deque<ChatPost> = []
+    @Published var pausedChatPosts: Deque<ChatPost> = []
```

Comment:
Making UI updates on every message received when chat is paused will give errors for busy chats (like 10+ messages/second). The 200 ms timer is used to make chat UI updates. I think we should keep this variable private and create a new published variable called `numberOfPausedChatPosts: Int = 0` that we update when the timer fires.

Reply:
great idea. have added a var pausedChatPostsCount